HUD/SetBlipDisplay.md

```diff
@@ -8,21 +8,23 @@ ns: HUD
 void SET_BLIP_DISPLAY(Blip blip, int displayId);
 ```
 
-**displayId Behaviour** <br>
-0 = Doesn't show up, ever, anywhere. <br>
-1 = Doesn't show up, ever, anywhere. <br>
-2 = Shows on both main map and minimap. (Selectable on map) <br>
-3 = Shows on main map only. (Selectable on map) <br>
-4 = Shows on main map only. (Selectable on map) <br>
-5 = Shows on minimap only. <br>
-6 = Shows on both main map and minimap. (Selectable on map) <br>
-7 = Doesn't show up, ever, anywhere. <br>
-8 = Shows on both main map and minimap. (Not selectable on map) <br>
-9 = Shows on minimap only. <br>
-10 = Shows on both main map and minimap. (Not selectable on map) <br>
-Anything higher than 10 seems to be exactly the same as 10. <br>
-<br>
+## displayId Behaviour
```

Comment:
Please, use `###` for custom headers. You can check the contribution guide [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)